proposals/smi.adoc

```diff
@@ -0,0 +1,123 @@
+== Service Mesh Interface Proposal
+*Name of Project:*
+Service Mesh Interface
+
+*Project Description:*
+Service Mesh Interface is an open source specification for interoperable service meshes on Kubernetes. Launched in May 2019, SMI brings the ecosystem together with a standard which enables innovation by providing a common abstraction that anyone can build against.
+
+*Statement on alignment with CNCF charter mission:*
+The Service Mesh Interface spec is well-aligned with the CNCF mission to make cloud native computing ubiquitous. Interoperability creates a virtuous cycle which increases adoption. SMI encourages an open source, vendor-neutral service mesh ecosystem by empowering end users to build upon ecosystem tools without needing to re-implement the underlying substrate.
+
+*Comparison with similar projects (inside or outside the CNCF), including what differentiates this project:*
+SMI is complementary to existing CNCF projects like LinkerD; it encourages adoption of service mesh. SMI is the only cross-industry effort to make service meshes interoperable. For example, LinkerD is one example of a service mesh that implements and uses SMI.
+
+*Sponsor from TOC (sponsor helps mentor projects):*
+TBD
+
+*Preferred maturity level (see CNCF Graduation Criteria):*
+Sandbox
+
+*License (charter dictates Apache 2 by default):* 
+The current license is OWF Contributor License Agreement 1.0 - Copyright and Patent. - https://github.com/deislabs/smi-spec/blob/master/CONTRIBUTING.md#specifications-subject-to-the-open-web-foundation-agreements
```

Comment:
Could we get a little background on the license choice?

Reply:
Disclaimer - IANAL.

This license was selected as it is generally used for industry specifications and covers the details for releasing and versioning specifications. This licensing is also adopted by the OCI as referenced in their charter documentations. Here is some reference material.
* [OCI v1.0.0 release notice](https://www.opencontainers.org/release-notices/v1-0-0)
* [OCI Governance docs](https://www.opencontainers.org/about/governance)
* [OWF CLA Copyright and Patent](http://www.openwebfoundation.org/legal/the-owf-1-0-agreements/owf-contributor-license-agreement-1-0---copyright-and-patent)

Reply:
Cool!  IANAL either... but I've walked enough things through legal to want to make sure we aren't making up our own licenses and are adopting best practices from others :)

Reply:
Appreciate the due-diligence and I agree with your statement about NOT "making" up our own licenses but rather adopting best practices.